### ISSUE TITLE
Permissions request triggered only by Activity

### ISSUE THREAD

**Issue by lukenowakcom** [GetStream/stream-chat-android#99]
When user opens attachment drawer in [https://github.com/GetStream/stream-chat-android/blob/master/library/src/main/java/com/getstream/sdk/chat/view/MessageInputView.java](url) it first asks for permission using only `Activity` context, while `Fragment` is passed as `null`
```java
@Override
    public void onClick(View v) {
        int id = v.getId();
        if (id == R.id.iv_send) {
            this.onSendMessage(binding.etMessage.getText().toString(), viewModel.isEditing());
        } else if (id == R.id.iv_openAttach) {
            binding.setIsAttachFile(true);
            PermissionChecker.permissionCheck((Activity) v.getContext(), null);
            messageInputClient.onClickOpenBackGroundView(MessageInputType.ADD_FILE);
        }
    }
```
It would be great to be able to pass `Fragment` to [https://github.com/GetStream/stream-chat-android/blob/master/library/src/main/java/com/getstream/sdk/chat/view/MessageInputView.java](url) so that the permissions request is triggered by the fragment itself.
Otherwise every permission request result needs to be handed down by activity to the fragment that handles chat.

**Comment by lukenowakcom** [comment]
Related to #78

**Comment by tbarbugli** [comment]
@lukenowakcom a fix for #78 is now available on master branch; we'll look into this issue as well
